apps/dialtone-documentation/docs/components/avatar.md

```diff
@@ -16,6 +16,41 @@ figma_url: https://www.figma.com/file/2adf7JhZOncRyjYiy2joil/DT-Core%3A-Componen
     </div>
 </code-well-header>
 
+<code-example-tabs
+htmlCode='
```

Comment:
The HTML part is a bit tedious and duplicative since we'd have manually create them. Maybe they can be auto-populated and prettified from the Vue example. That might get a bit long and eliminates how we've intentionally collapsed some of the HTML, e.g. `<svg>...</svg>`. Maybe some of those truncated/collapsed parts can be identified and ripped out at build.

Maybe a stretch goal or future task.

Reply:
Yeah thought about this a bit myself. not really that easy of a problem to solve, definitely future goal